api/envoy/extensions/filters/http/grpc_stats/v3/config.proto

```diff
@@ -54,6 +54,11 @@ message FilterConfig {
     //   `envoy.deprecated_features.grpc_stats_filter_enable_stats_for_all_methods_by_default`.
     google.protobuf.BoolValue stats_for_all_methods = 3;
   }
+
+  // If true, the filter will gather a histogram for the request time of the upstream.
```

Comment:
Please make the references to other fields be links. Example from elsewher: 
```
:ref:`private_key <envoy_api_field_extensions.transport_sockets.tls.v3.TlsCertificate.private_key>`
```

Reply:
Should I change the doc to point to v3 then also?